src/calculateRentalCost.js

```diff
@@ -4,7 +4,14 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
-}
+  let result = days * 40;
```

Comment:
Use `const` instead of `let` for `result` if you're not planning to reassign it. In this case, you can calculate the total cost and apply discounts in separate steps.